README.md

```diff
@@ -6,17 +6,18 @@
 Nothing but net(working).
 
 Swish is a networking library that is particularly meant for requesting and
-decoding JSON via [Argo](http://github.com/thoughtbot/Argo). It is protocol
-based, and so aims to be easy to test and customize.
+decoding JSON via `Decodable`. It is protocol based, and so aims to be easy
+to test and customize.
 
 ## Version Compatibility
 
 Here is the current Swift compatibility breakdown:
 
-| Swift Version | Swish Version |
-| ------------- | ------------ |
-| 3.X           | 2.X          |
-| 2.X           | 1.X          |
+| Swift Version | Swish Version  |
+| ------------- | ------------   |
+|           4.X | >= 2.0.3       |
```

Comment:
Should we bump the major version for this change?

Reply:
Definitely!